rescript.opam

```diff
@@ -10,6 +10,7 @@ depends: [
   "ocaml" {>= "4.10"}
   "ocamlformat" {= "0.26.2"}
   "cppo" {= "1.6.9"}
+  "js_of_ocaml" {= "5.8.1"}
   "js_of_ocaml-compiler" {= "5.8.1"}
```

Comment:
I think js_of_ocaml depends on js_of_ocaml-compiler, so it should be sufficient to just have js_of_ocaml in here.